.github/actions/gradle-setup/action.yml

```diff
@@ -3,41 +3,41 @@ description: "Setup Gradle"
 runs:
   using: "composite"
   steps:
-    # Checkout EDC code into DataSpaceConnector directory.
```

Comment:
why this change? Though both forms are valid, it is quite more common in my experience to indent there (also in GH doc)

Reply:
As mentioned above, this file will be restored as soon as https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1792#pullrequestreview-1065206083 gets merged